client/src/components/Home.js

```diff
@@ -5,9 +5,36 @@ import { Header } from "semantic-ui-react";
 const Home = () => {
   return (
     <div>
-      <header className="home-header">
-        <h1>Broca Translation</h1>
-      </header>
+      <Header
+        as="h1"
+        textAlign="center"
+        content="Broca Translation"
+        image="http://www.24hourtranslation.com/wp-content/uploads/2010/04/global-translation.jpg"
+      ></Header>
+
+      <img
+        src="https://blogsensebybarb.files.wordpress.com/2014/03/happiness9-aristotle.jpg"
+        className="homePageImg"
+      />
+      <div className="information">
+        <p className="land-p-one">
+          An app to create a user-based network of volunteer
+          interpreters/translators. Translators will be able to create profiles
+          with the language they speak.
+          <br /> The History Project (which is part of a real organisation) can
+          upload chosen archival pieces to the app and requests their
+          translation from volunteer interpreters.
+        </p>
+        <p className="land-p-two">
+          They can also upload invites and other texts for translation and
+          distribution. As a stretch goal, Users can also use the app as a
+          platform to upload texts or photos of texts in another language that
+          they do not understand, including what we commonly nickname
+          “legalese”.
+        </p>
+        <button className="signUp">Sign Up</button>
+        <button className="interpreter">Become an Interpreter</button>
+      </div>
```

Comment:
Can you use primary button from Semantic UI here? So we have consistent styling. Check this here: https://react.semantic-ui.com/elements/button/#types-emphasis. You can remove the signUp and interpreter styles in the css after.

Reply:
I've used it already but the issue is that the locations are in undesired places and could not be modified by css as it is overridden by semantic.

Reply:
You can pass a className to the Semantic UI buttons, which center the buttons. Alternatively, you could also wrap these buttons into a div that you center on the screen.